src/pages/taking-koselugo.astro

```diff
@@ -1,18 +1,127 @@
+            <span>Not liking the taste or texture</span>
+          </li>
+          <li class="taking-koselugo__list-item">
+            <img class="taking-koselugo__image swallowing-tablets-icon" src="/assets/images/taking-koselugo/swallowing-tablets-icon.png" alt="">
+            <span>Having difficulty swallowing medication</span>
+          </li>
+        </ul>
+        <p>It’s important that your child understands why it’s necessary to take 2 doses of Koselugo every day. Be sure to talk about it in a way that’s positive. And, as your child grows, you may need to change your approach to ensure that the dosing schedule is followed.</p>
+        <h6 class="taking-koselugo__title">Suggestions to help give medicine to young children:</h6>
+        <ul class="taking-koselugo__list">
+          <li class="taking-koselugo__list-item">Role playing can be helpful: Pretending to give a favorite doll or teddy bear some medicine can make taking medicine more fun</li>
+          <li class="taking-koselugo__list-item">Try to do a positive, fun activity after your child takes the medicine. This may help motivate your child to take the medicine again</li>
+        </ul>
+        <p>
+          <b>For some helpful tips to teach your child how to swallow pills,</b> you may want to download the brochure
+          <a href="https://ccr.cancer.gov/sites/default/files/medbooklet.pdf#search=swallowing" target="_blank" class="text--decoration-underline color--rose text--italic">Medicine & Your Child: A Guide for Parents on Adherence and Administration</a> or visit <a href="https://kidshealth.org/en/parents/swallowing-pills.html" target="_blank" class="text--decoration-underline color--rose">www.kidshealth.org</a> to see <span class="text--italic">Teaching Your Child How to Swallow Pills</span> for additional suggestions.
+        </p>
+        <p class="taking-koselugo__footnote">Please note: These links will take you to sites that are maintained by a third party who is solely responsible for its contents. Alexion is not responsible for the privacy policy of any third-party websites. We encourage you to read the privacy policy of every website you visit.</p>
+      </Section>
+      <Section className="taking-koselugo__schedule">
+        <Fragment slot="section-title">
+          HOW CAN I ESTABLISH A DAILY SCHEDULE FOR <span class="color-yellow">TAKING KOSELUGO?</span>
+        </Fragment>
+        <p class="taking-koselugo__paragraph">Work with your child’s doctor to establish a dosing routine that can be easily followed. Taking each dose of Koselugo as directed, every day, is the best way to benefit from treatment.</p>
+        <picture class="taking-koselugo__image-container">
+          <source media="(min-width: 725px )" srcset="/assets/images/taking-koselugo/schedules-for-dose-desktop.png">
+          <img class="taking-koselugo__image schedules-for-dose" src="/assets/images/taking-koselugo/schedules-for-dose-mobile.png" alt=" ">
```

Comment:
This asset shouldn't be wrapped.

## Code
![image](https://user-images.githubusercontent.com/31933833/229160282-eebb04b0-c6ba-4695-8aa0-f27d0981c282.png)

## Figma
![image](https://user-images.githubusercontent.com/31933833/229160387-0e1f315d-7e5a-4920-8f19-fd948432aae1.png)